### DISCUSSION TITLE
Is it possible to search open browser tabs

### DISCUSSION THREAD

**Discussion by mx3214** [Flow-Launcher/Flow.Launcher#2344] [Q&A] [answered]
Hi,

Not sure if this is possible natively or via a plugin (eg window walker) in flow launcher. But can one search open browser tabs, especially across multiple windows in eg Edge browser?

Thanks

**Comment by jjw24** [comment, marked as answer]
Hey there, this is a feature we would like for WindowWalker and haven't really found a good way to do it yet.
